launcher/src/main/java/org/apache/spark/launcher/package-info.java

```diff
@@ -45,5 +45,33 @@
  *   }
  * }
  * </pre>
+ *
+ * <p>
+ * Since Spark 1.5.0, the {@link org.apache.spark.launcher.SparkLauncher#startApplication(
```

Comment:
I think this should go above the section on `launch`, the `launch` section should clearly state its the old way of doing things.